files/lmod

```diff
@@ -1 +1,5 @@
-LMOD_MSG (?:- lmod::)\s*user=%{USERNAME_PAT}, cluster=%{DATA:cluster}, arch=%{DATA:architecture}, module=%{DATA:module_name}/%{DATA:module_version}, fn=%{DATA:module_path}, jobid=%{DATA:jobid}$
+LMOD_GENERAL (?:- lmod::)\s*username=%{USERNAME_PAT}, cluster=%{DATA:cluster}, arch=%{DATA:architecture}, jobid=%{DATA:jobid}
+LMOD_USERLOAD (?:yes|no)
+LMOD_LOAD userload=%{LMOD_USERLOAD:userload}, module=%{DATA:module_name}/%{DATA:module_version}, fn=%{GREEDYDATA:module_path}
+LMOD_CMD cmd=%{DATA:command}, args=%{GREEDYDATA:arguments}
+LMOD_MSG %{LMOD_GENERAL}, (?:%{LMOD_LOAD}|%{LMOD_CMD})
```

Comment:
lgtm, but I probably wouldn't be able to recognise mistakes here...

maybe @itkovian or @alvarosimon shoul take a look at this too?